imagersite/imagersite/tests.py

```diff
@@ -0,0 +1,38 @@
+from django.test import TestCase
+from django.test.client import Client
+from django.contrib.auth.models import User
+from django.urls import reverse_lazy
+
+
+class ViewTests(TestCase):
+    """test class that tests login view"""
+    @classmethod
+    def setUp(cls):
+        User.objects.create_user(username='testuser', password='SecretPassword')
+
+    def test_login(self):
+        """testing that user autheticated correctly after login"""
+        # send login data
+        response = self.client.post(
+            reverse_lazy('auth_login'), 
+            {'username': 'testuser', 'password': 'SecretPassword'},
+            follow=True,      
+            )
+        # user should be logged in.
+        self.assertTrue(response.context['user'].is_authenticated)
+        self.assertRedirects(response, reverse_lazy('user_profile'))
```

Comment:
You may also want to have a tests that checks for failure. For example, if the user gives the wrong password, they do not get logged in.